include/rcutils/env.h

```diff
@@ -55,6 +55,24 @@ RCUTILS_WARN_UNUSED
 bool
 rcutils_set_env(const char * env_name, const char * env_value);
 
+/// Set or un-set a process-scoped environment variable while specifying overwrite behavior.
+/**
+ * Documentation inherited from rcutils_set_env.
```

Comment:
Move the detailed description of rcutils_set_env here, as the function will eventually be removed.